Tryouts/Core/Abstractions/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. -->
+
+# MorganStanley.ComposeUI.Tryouts.Core.Abstractions
+===================================================
+
+Abstractions project contains interfaces and simple classes/structures that they depend on. These interfaces are being implemented primarily by the dynamically
+loaded plugins and are used for communications between the varoius plugins, and the shell.     
```

Comment:
I don't think these abstractions are restricted to just communication between the various modules. Communication is one aspect that Compose would provide but there are many others.

Reply:
I meant 'communications' in a broader sense than just messaging. I changed it to 'interactions' - hopefully it is a better term.